crates/ruma-identity-service-api/src/invitation/store_invitation/v2.rs

```diff
@@ -106,7 +107,42 @@ impl<'a> Request<'a> {
 
 impl Response {
     /// Creates a new `Response` with the given token, public keys and display name.
-    pub fn new(token: String, public_keys: Vec<String>, display_name: String) -> Self {
+    pub fn new(token: String, public_keys: PublicKeys, display_name: String) -> Self {
         Self { token, public_keys, display_name }
     }
 }
+
+/// The server's long-term public key and generated ephemeral public key.
+#[derive(Debug, Clone)]
+pub struct PublicKeys {
```

Comment:
I think we can safely assume that the field will always be an array of two elements, not more so this struct can be exaustive (`#[allow(clippy::exhaustive_structs)]`).

Reply:
If it's non-exhaustive and doesn't have a constructor, users won't be able to construct this. So there either needs to be a constructor, the non_exhaustive attr has to be removed again, with the aforementioned allow attribute added.